ansible/playbooks/AdoptOpenJDK_ITW_Playbook/roles/Common/scripts/install-xcode.sh

```diff
@@ -1,10 +1,14 @@
 #!/bin/bash
+os_maj_vers=$(sw_vers -productVersion | awk -F "." '{print $1}')
 osx_vers=$(sw_vers -productVersion | awk -F "." '{print $2}')
 cmd_line_tools_temp_file="/tmp/.com.apple.dt.CommandLineTools.installondemand.in-progress"
 
+echo 1 = $os_maj_vers
+echo 2 = $osx_vers
+
 # Installing the latest Xcode command line tools on 10.9.x or higher
 
-if [[ "$osx_vers" -ge 9 ]]; then
+if [[ "$os_maj_vers" -ge 10 ]] && [[ "$osx_vers" -ge 9 ]]; then
```

Comment:
Should this be `-eq 10` for the first clause?

Reply:
Yup, as best I can tell ITW is restricted to OSX 10.x , fixed.